templates/ClientMethodOptionsFile.stencil

```diff
@@ -0,0 +1,7 @@
+{% include "HeaderSnippet.stencil" %}
+
+{% include "ImportSnippet.stencil" %}
+
+{% for model in models %}
+    {% include "ClientMethodOptionsSnippet.stencil" model %}
```

Comment:
Cool. I did not know you can pass in parameter when include another stencil.

Reply:
Yes--quite a useful feature! It means we can get away with fewer ViewModel structs 😁 
https://stencil.fuller.li/en/latest/builtins.html#include